### PR TITLE
Enabled 6 more ELF enclaves for Windows by reducing enclave size

### PR DESCRIPTION
**Pull request by anitagov**
Enabled 6 more ELF enclaves tests on Windows by resizing enclave sizes (#1008 ). These are: backtrace, ecall-ocall, stdcxx, thread, threadcxx, thread_local.

### CODE REVIEW

**Comment by anitagov** [comment]
bors try

**Comment by anitagov** [comment]
bors try

**Comment by anitagov** [comment]
bors try

**Comment by anitagov** [comment]
bors try

**Comment by anitagov** [comment]
bors try

**Comment by anitagov** [comment]
bors try

**Comment by anitagov** [comment]
bors try

**Comment by anitagov** [comment]
@anakrish @johnkord @yakman2020 @soccerGB - Need one of you to review and accept this PR ASAP so that we have more tests passing on Windows CI.

**Comment by anitagov** [comment]
Still awaiting review

File: CMakeSettings.json [hunk 1]
```diff
@@ -67,14 +67,14 @@
     },
     {
       "name": "x64-Debug-tests",
-      "generator": "Ninja",
+      "generator": "Visual Studio 15 2017 Win64",
```
Comment (anakrish) [inline CMakeSettings.json]: I don't see how these changes are related to the PR.

Comment (anitagov) [reply CMakeSettings.json]: Roped in changes made by VVDN from PR #1107 . This section is for WSL built enclave tests running under Visual Studio 2017 and not exercised by Windows CI.

**Pull request merged**